shortlister/model.py

```diff
@@ -310,3 +310,41 @@ def abbreviate(list_of_strings: List[str]) -> list[str]:
         else:
             abbreviations.append(string)
     return abbreviations
+
+# filter functions
+def name(applicant:Applicant, name:str):
+    """Filter by matching applicant name.
+    Example usage:  name(applicant,"Emma")"""
+    return name.lower() in applicant.name.lower()
+
+def score(applicant:Applicant,name,score):
+    """Filter by matching applicant score.
+    Example usage:  score(applicant,"PhD","Excellent")"""
+
+    # checks that (criterion) name does not match any criterion in applicant scores
+    if score == None:
+        return name.lower() not in [getattr(criterion,"name").lower() for criterion in applicant.scores]
+    
+    # checks if (criterion) name and score matches the saved applicant scores
+    for criterion in applicant.scores:
+        if getattr(criterion,"name").lower() == name.lower():
+            return applicant.scores[criterion].lower() == score.lower()
+        else:
+            continue
```

Comment:
You don't need this - the loop will automatically continue if the criterion doesn't match.